naming/src/main/java/com/alibaba/nacos/naming/core/ServiceManager.java

```diff
@@ -1135,6 +1140,49 @@ public void run() {
         }
     }
     
+    private static class ServiceNameHelper implements Comparable<ServiceNameHelper> {
```

Comment:
The meaning of this class? I feel that with this design, I am very puzzled. Is there a Map inside ServiceNameHelper?